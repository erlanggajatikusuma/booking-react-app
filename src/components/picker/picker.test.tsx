import React from 'react';
import renderer from 'react-test-renderer';
import {Picker} from './Picker';

describe('Picker Component', () => {
  it('renders correctly', () => {
    const items = [
      {label: 'Option 1', value: 'value1'},
      {label: 'Option 2', value: 'value2'},
    ];

    const tree = renderer
      .create(
        <Picker
          items={items}
          selectedValue={'value1'}
          onValueChange={() => {}}
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
