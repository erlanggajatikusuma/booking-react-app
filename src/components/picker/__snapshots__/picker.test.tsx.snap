// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picker Component renders correctly 1`] = `
<View
  style={
    [
      undefined,
      {
        "height": 42,
        "width": 100,
      },
    ]
  }
>
  <RNCPicker
    items={
      [
        {
          "label": "Option 1",
          "testID": undefined,
          "textColor": undefined,
          "value": "value1",
        },
        {
          "label": "Option 2",
          "testID": undefined,
          "textColor": undefined,
          "value": "value2",
        },
      ]
    }
    numberOfLines={1}
    onChange={[Function]}
    selectedIndex={0}
    style={
      [
        {
          "height": 216,
        },
        {
          "height": 42,
          "width": 100,
        },
      ]
    }
  />
</View>
`;
